main/main.cpp

```diff
@@ -654,6 +655,7 @@ void Main::print_help(const char *p_binary) {
 	print_help_option("-s, --script <script>", "Run a script.\n");
 	print_help_option("--main-loop <main_loop_name>", "Run a MainLoop specified by its global class name.\n");
 	print_help_option("--check-only", "Only parse for errors and quit (use with --script).\n");
+	print_help_option("--format", "Format the file in-place or (if used with --check-only) check if file is formatted correctly. This option must be used with --script and GDScript file.\n");
```

Comment:
```suggestion
	print_help_option("--format", "Format the script in-place (use with --script) or check if the file is formatted correctly (use with --script --check-only).\n");
```

Reply:
I wonder about CI, if we would want a "format to stdout". And the help option should be preceded by `[EXPERIMENTAL]` imho.

Reply:
> I wonder about CI, if we would want a "format to stdout". And the help option should be preceded by `[EXPERIMENTAL]` imho.

As for the "format to stdout" - IMO we can add it on-demand (when and if required) once we have some actual formatting implemented. Chances are this won't be ever needed.

As for the `[EXPERIMENTAL]` can you elaborate? Or do you just mean smth like `print_help_option("--format", "[EXPERIMENTAL] Format the script(...)`?